src/views/product-integration/component-view/components/variable-group-list/index.tsx

```diff
@@ -1,12 +1,24 @@
 import { IconCornerDownRight16 } from '@hashicorp/flight-icons/svg-react/corner-down-right-16'
 import classNames from 'classnames'
 import Badge from 'components/badge'
-import ReactMarkdown from 'react-markdown'
+import DevDotContent from 'components/dev-dot-content'
+import { MdxInlineCode, MdxP } from 'components/dev-dot-content/mdx-components'
+import { MDXRemoteSerializeResult } from 'next-mdx-remote'
 import s from './style.module.css'
 
+/**
+ * We use some custom elements to decrease the font size for paragraph
+ * content when rendering processed MDX variable descriptions.
+ */
+const smallDescriptionMdxComponents = {
+	inlineCode: (props) => <MdxInlineCode {...props} size="100" />,
+	p: (props) => <MdxP {...props} size="200" />,
+}
+
 export interface Variable {
 	key: string
 	type: string
+	descriptionMdx?: MDXRemoteSerializeResult
 	description?: string
```

Comment:
Should this be removed now?

Reply:
Ah good callout, I needed to add comments here to clarify similar to the one in `component-view/index.tsx` - `descriptionMdx` is used to render the description visibly, while the plain `description` string is used for searching and filtering purposes 👍